frontend/src/__tests__/cypress/cypress/pages/enabled.ts

```diff
@@ -1,6 +1,6 @@
 class EnabledPage {
-  visit(homeEnabled = false) {
-    cy.visit(homeEnabled ? '/enabled' : '/');
+  visit() {
+    cy.visit('/enabled');
```

Comment:
I don't understand why you changed this... the home page can still be disabled

Reply:
I could add it back but it's not set in any of the callers.

Reply:
You had a test to make sure the URL redirect was properly tested with and without the feature enabled. I'm feeling you had to have dropped something or we are missing something if it's no longer used.

This cannot be right I would think 🤔 Not sure where the gap is...

Reply:
I changed the test to verify that `/` takes you to the enabled page if the flag is disabled:
https://github.com/opendatahub-io/odh-dashboard/pull/2767/files#diff-34ec9e23c634748dc1ed7b69552b385bf9a11a5a746b7e4779c48e3302f9699aR19